jibri-xmpp-client/jibrixmppclient.py

```diff
@@ -322,11 +334,13 @@ def report_jibri_error(self, error):
         except Exception as e:
             logging.error("Failed to send status update: %s", str(e))
 
-    def update_jibri_status(self, status):
+    def update_jibri_status(self, status, sipaddress):
```

Comment:
could provide a default value for sipaddress here so that it doesn't require the parameter in all calls